src/lib.rs

```diff
@@ -21,6 +22,16 @@ pub fn cpu_temperature(sys: &System) -> f32 {
 }
 
 /// Returns the command line of top CPU heavy processes.
```

Comment:
nit(pick): It would be nice if the return value was a little more clear to use; you could either make the tuple provided into a struct with descriptive fields or change the documentation of the function to clarify what each element of the tuples in the Vec represent.

Reply:
Good idea. I added a new ProcessStruct